SimplifiedTests/NYPLAxisTests/license-9781250109699.json

```diff
@@ -0,0 +1 @@
+{"id":"402d26ea-5213-4f40-848e-55ca61c04802","version":"1","issued":"2021-05-12T15:06:36.263Z","provider":"Baker And taylor","encryption":{"content_key":{"algorithm":"http://www.w3.org/2000/09/xmldsig#rsa-sha1","encrypted_value":"iLpuYR02lOPlpcAwRI3BpNH5l9uCzDGe9OS4TlNc5fohDWrGlUcQOD6+VE4ruB8qQQs4vUgVDIes35kidzYyI+fZ4l7mG8bMF4vBthViaG/AbVFMOl1EmKdDrTuTPGEsi6hTspx/zxTjEHC8t6WfMRhpv4GHoQkcnlcKTki9lgTzr3Dd9r28wS+fbBlbMqrEre+QT5i6bz+OokJhZY4EIrJITxNIVgXfLkJVBt+UVcQ4LBG088Ol1d6OqBqdEt5zqi0y+tcMnXWH3RFYGZ27I3z6h16SIBJEXfXv3Ev7tmjFK+O/OKkR9nkF2NW3lQ/MPDnTGGIpon93N8AC5M++Yw=="},"profile":"http://www.w3.org/2000/09/xmldsig#rsa-sha1","user_key":{"algorithm":"http://www.w3.org/2000/09/xmldsig#rsa-sha1","key_check":"tndJkNNvkaniVKKRU1uMKi0sNGmKL2iq7GVuABz6KxYM1pcxf+nF8aFWssIg0iFs9fw+/STdahmatquPY8jeo9pS68UwpZu2oM735KEklEGFEvg9diI1ckIsvnuZCIUMV5TYgMddKbMgED6SA5ovH8Q0KvE2NNP5s6bWZFDugbbhvudSTpXl/JdvRqKUcmhyIAMWWrmjLJRjzgLl66SZGsAQr+7ugT/m6FfgPEEv/O1tZ2zrMtR3pMpa1av0kPyD1prlfP92Kwruh2hheAD+weOb/TzokrvQ/bhqqdEM96ZzbyUpNoIwp4uWK02v/1yvQx3s/jrw1b1HRJmWak9fpw==","text_hint":""}},"links":[{"hint":{"href":""},"publication":{"href":""}}],"rights":{"copy":0,"end":"2021-07-07T15:06:00.000Z","print":0,"start":"2021-05-12T15:06:36.263Z","tts":false},"drmrights":{"copy":{"copyright":true,"days":365,"notice":"Copyright held by Publisher.  All rights Reserved.","percentage":50,"permission":false,"watermark":true},"devices":6,"preview":null,"print":{"copyright":false,"days":5,"notice":"","percentage":50,"permission":false,"watermark":false},"reflow":true,"ttsread":true},"signature":{"algorithm":"http://www.w3.org/2000/09/xmldsig#rsa-sha1","certificate":"BgIAAACkAABSU0ExAAgAAAEAAQAlOqGroR57awwqgtZgPCIrT2+fwq3bxzyF9HSDghHxaqECT2Aaf60c0usWGAJRbi00fS5OYoCox753bc6w4Ii8F7OooNuqTFfILUk1qDcyDoMKRVjuPf1oDcwmygt/CSnbngUDCtrLmTGUxzjac8KSIZ3mGBVBPr+FV9hBLQhWcv4srhlHpOvwQQGR4aftRyi2hUhJWqECqVGqxY5cp94pRnlYiOy0GCFCYjkQI9T+wEbBKHSGQsabYTJO2u51wDhgYZEL+t0ec6F/F6i/zfGI+hBxFSnZfgj/NhcNvjDOho2YCIGGfH0Cv4xz3cCcj8EQOES9n9PwjVKq9/Ai8ifM","value":"I7Ydnew5Yc5KmCyxoky5qXnTVqGGt54dxsZ4g00dlbGLSxrgV7tuTLskRn/3n8ywRDZZDt9VnAaMQDDRyQOlVONm/G9yQcHr0PxL7U+lGDhyHUIYKa5VA27uUEe2UB37+YZmU8I68nppT+hgq1aqRpcDZtud6JOIQK2CyMjxxjA71f7mMAHs+IxwM8NCKISIS/IRpxXOwc9U5V/5ygZgo1xEqoVlw/CXnXy7D1RBser2NUqRWYVnlDkIAfE6mtE4ml2aO9eEdV0uGpLLU+5XZTlq6YPcmFv4MxPz6BkOmMHzEpoGC1g5/3/oPlBmd6AyKobXM3D1ZBQ3f59rLAudbQ==","LicenseConicalForm":null,"RawHashLicenseConicalForm":null,"RawAESkey":null}}
```

Comment:
yes these are. To download and decrypt a book, one needs the bookId and bookVault id. The license contains bookVaultID but it's encrypted with the key we sent before downloading the license. Since there is no bookId and the bookVaultID in the license is encrypted with our private key, nobody can have unauthorized access to the books.

Reply:
That's good, however it's still a hacking vector for a malicious person: this might be ok for us or it might not. I've definitely be in situations where this would not be ok, so I should ask if this was cleared by Risa aka @killerpuppytails and how comfortable we are with this. There might be legal requirements I'm not aware of.

Reply:
Modified license file. None of the values in the license are real anymore. The base64 string is no longer pointing to the key.

Reply:
So this flow wasn't cleared by me, no, but this is a known B&T process. Let's bring @leonardr into this conversation, because I believe there's a way to make this more secure.

Reply:
In a situation like this, masking the real values is the right call. Since the old values are now under version control, it would also be good to rebase this branch so they don't get merged into the history. Based on my knowledge of AxisNow, I agree with Raman that these values don't themselves compromise security, but it's best to be conservative.

Reply:
@ettore @leonardr @killerpuppytails the branch has been rebased with masked values.